js/host/main.js

```diff
@@ -98,7 +98,32 @@ const nlwManager =
   new NLWManager( byEID("nlw-frame"), connMan.broadcast
                 , connMan.narrowcast, onNLWManError);
 
-document.addEventListener("DOMContentLoaded", nlwManager.init);
+document.addEventListener("DOMContentLoaded", () => {
+  const modalContainer = byEID("modal-container");
+
+  window.onclick = (event) => {
+    if (event.target === modalContainer) {
+      modalContainer.classList.toggle("hidden");
```

Comment:
I'm pretty sure I used the word "toggle" when I suggested this set of changes to you, but I chose my words poorly.  I'm a real stickler about ambiguity.  I hate having to think when I read code.  I would prefer that we were explicit in each of these `classList` modifications.  `classList.toggle` is almost never what I actually want, and, instead, I prefer `classList.add` and `classList.remove`.

Sorry for being confusing.

Reply:
No worries at all - this is a super easy fix. Done!